databricks/koalas/frame.py

```diff
@@ -3698,7 +3700,121 @@ def notnull(self) -> "DataFrame":
 
     notna = notnull
 
-    # TODO: add frep and axis parameter
+    def insert(
+        self,
+        loc: int,
+        column,
+        value: Union[Scalar, "Series", Iterable],
+        allow_duplicates: bool = False,
+    ) -> None:
+        """
+        Insert column into DataFrame at specified location.
+
+        Raises a ValueError if `column` is already contained in the DataFrame,
+        unless `allow_duplicates` is set to True.
+
+        Parameters
+        ----------
+        loc : int
+            Insertion index. Must verify 0 <= loc <= len(columns).
+        column : str, number, or hashable object
+            Label of the inserted column.
+        value : int, Series, or array-like
+        allow_duplicates : bool, optional
+
+        Examples
+        --------
+        >>> kdf = ks.DataFrame([1, 2, 3])
+        >>> kdf
+           0
+        0  1
+        1  2
+        2  3
+        >>> set_option("compute.ops_on_diff_frames", False)
```

Comment:
Seems `set_option` is not imported somewhere above.

Reply:
Good catch! Thanks.